activerecord/test/cases/adapters/mysql2/mysql2_adapter_test.rb

```diff
@@ -262,6 +262,13 @@ def test_statement_timeout_error_codes
     end
   end
 
+  def test_doesnt_error_when_a_use_query_is_called_while_preventing_writes
+    @connection_handler.while_preventing_writes do
+      db_name = ActiveRecord::Base.configurations["arunit"][:database]
+      @conn.execute("USE #{db_name}")
```

Comment:
We should try to always assert something when writing a test, otherwise it could silently fail. I did look at this locally and this returns a nil instead of an ActiveRecord::Result. Given that this returns nil we have nothing to assert on except that it doesn't raise.

Can you add an assertion that it doesn't raise so we're at least asserting something?

Reply:
Do you mean to wrap it in `assert_nothing_raised` block similar to https://github.com/rails/rails/blob/a8d5e37dfc94c0695fb8da8e53172814161805cc/activerecord/test/cases/adapters/mysql2/mysql2_adapter_test.rb#L15-L20?

Reply:
Yes